Makefile

```diff
@@ -1,6 +1,6 @@
 PACKAGES=$(shell go list ./... | grep -v '/simulation')
 PACKAGE_NAME:=github.com/crypto-org-chain/chain-main
-GOLANG_CROSS_VERSION  = v1.16.3
+GOLANG_CROSS_VERSION  = v1.17.3
```

Comment:
the latest Go 1.17 is 1.17.13 which had some security fixes -- can that be used instead?